README.md

```diff
@@ -320,6 +320,9 @@ Pantheon Advanced Page Cache integrates with WordPress plugins, including:
 See [CONTRIBUTING.md](https://github.com/pantheon-systems/pantheon-advanced-page-cache/blob/master/CONTRIBUTING.md) for information on contributing.
 
 ## Changelog ##
+### 1.4.2 ###
```

Comment:
I like having the date, I know it's been missed the last couple versions and might be more difficult with CCB. Optional.

Reply:
Hmm...I guess we can add it back in last minute following CCB approval?